hack/lib/images.bash

```diff
@@ -235,6 +245,21 @@ function latest_konflux_image_sha() {
   echo "${image_without_tag}@${digest}"
 }
 
+function latest_registry_ci_sha() {
+  input=${1:?"Provide image"}
+
+  image_without_tag=${input%:*} # Remove tag, if any
+
+  go_bin="$(go env GOPATH)/bin"
+  export GOPATH="$PATH:$go_bin"
+  digest=$(skopeo inspect --no-tags=true "docker://${input}" | jq -r '.Digest')
+  if [ "${digest}" = "" ]; then
+    exit 1
+  fi
+
+  echo "${image_without_tag}@${digest}"
+}
```

Comment:
I think we could reuse `latest_konflux_image_sha` for this ?

Reply:
I will check this. We don't want the "latest" tag though. We want a specific tag, e.g. release-1.34.0. The `latest_konflux_image_sha` automatically adds "latest"

Reply:
Pushed.